### DISCUSSION TITLE
Need help with LXC mounting NFS share from TrueNAS

### DISCUSSION THREAD

**Discussion by sgtjamez** [tteck/Proxmox#3377] [Help Needed]
Mittens and I ran afoul of another issue... As part of the OpenWRT/Arr YouTube series, Novaspirit is setting up the "Arr" suite (Prowlarr/Sonarr/Radarr). He mounted an NFS share from his PVE host. My NFS share is on a TrueNAS box and I am having permissions issues. What I don't know how to do is bind permissions.

My PVE setup to mount the share to Proxmox:
```
nfs: downloadz
        export /mnt/main_vault/app_storage/qbit_dl
        path /mnt/pve/downloadz
        server 192.168.40.3
        content vztmpl
        prune-backups keep-all=1
```
My command to mount the NFS share to Sonarr
`pct set 306 --mp0 /mnt/pve/downloadz,mp=/mnt/media`

The share is accessible from the LXC console, but does not appear in the Sonarr System page. I believe it to be a permissions issue but I don't know how to fix it.

Let's say user TrueNAS user 'apps' (UUID 568 I believe) has full control of the share, how do I assign that to the PVE side of the equation?

**Comment by austinlane72** [comment]
I'm also using a NFS share from TrueNAS and am having the same problem. I managed to get access in the LXC by creating a group with the same GID as the group owning the share on TrueNAS.

Did you ever find a solution for this?
